assemble/bin/start-daemon.sh

```diff
@@ -103,6 +103,9 @@ if [[ "$SERVICE" != "tserver" || $NUM_TSERVERS -eq 1 ]]; then
    fi
    echo "Starting $SERVICE on $HOST"
 
+   OUTFILE="${ACCUMULO_LOG_DIR}/${SERVICE}_1_${LOGHOST}.out"
```

Comment:
Actually, this isn't right. The call to `rotate_log` on lines 90,91 would get lost.